spec/order_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'order'
+
+describe Order do
+  let(:menu) { double :menu }
+  let(:subject) { described_class.new(menu) }
```

Comment:
### Using `subject` instead of `let`

Use the [explicit `subject`](https://www.relishapp.com/rspec/rspec-core/docs/subject/explicit-subject) here instead of `let`, as you're modifying the implicit `subject` rather than declaring something wholly new.